### PR TITLE
fix: link as font needn't replace

### PR DESCRIPTION
**Pull request by shuyanzi**

### CODE REVIEW

File: src/__tests__/test-process-tpl.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ test('test process-tpl', () => {
 		'<link rel="preload" href="//gw.alipayobjects.com/as/g/antcloud-fe/antd-cloud-nav/0.2.22/antd-cloud-nav.min.js">\n' +
 		'<link rel="prefetch" href="/a3-ie6-polyfill.js">\n' +
 		'<link rel="stylesheet" href="/umi.css">\n' +
+		'<link rel="preload" as="font" href="/static/fonts/iconfont.woff" type="font/woff" crossorigin="anonymous">\n' +
```
Comment (kuitos) [inline src/__tests__/test-process-tpl.js:16]: need to check font not be replaced after processed

File: src/process-tpl.js [hunk 2]
```diff
@@ -88,7 +89,13 @@ export default function processTpl(tpl, baseURI) {
...
 			}
-
+			/*
+			as font normal
+			*/
+			const asFont = !!match.match(LINK_AS_FONT);
+			if (asFont) {
+				return match;
+			}
 			const preloadOrPrefetchType = match.match(LINK_PRELOAD_OR_PREFETCH_REGEX) && match.match(LINK_HREF_REGEX);
```
Comment (kuitos) [inline src/process-tpl.js:99]: combine `asFont` condition here, such as
```js
const preloadOrPrefetchType = match.match(LINK_PRELOAD_OR_PREFETCH_REGEX) && match.match(LINK_HREF_REGEX) && !!match.match(LINK_AS_FONT)
```

File: src/__tests__/test-process-tpl.js (hunk 1 above)
Comment (shuyanzi) [reply src/__tests__/test-process-tpl.js:16]: yes, I checked it by ‘console.log(template)‘

File: src/process-tpl.js (hunk 2 above)
Comment (shuyanzi) [reply src/process-tpl.js:99]: haha 😄, I did this last commit, I'll change back

**Comment by kuitos** [comment]
ci failed

File: src/__tests__/test-process-tpl.js (hunk 1 above)
Comment (kuitos) [reply src/__tests__/test-process-tpl.js:16]: Should be checked with an assert code rather than a manual work

**Comment by shuyanzi** [comment]
> ci failed

done!

I'm so careless!sorry~

**Pull request merged**